### ISSUE TITLE
Tooltip formatting after introduction of PrimeReact

### ISSUE THREAD

**Issue by jreineckearm** [eclipse-cdt-cloud/vscode-peripheral-inspector#29]
Type: Bug Report

**Describe the bug**
- OS and Version: Windows 11 Enterprise
- VS Code Version: 1.93.1
- Extension Version: Local build of `main` at https://github.com/eclipse-cdt-cloud/vscode-peripheral-inspector/commit/61550b2e2a83c8a59bd6183ee317ca4922cb2758
- Target Device:
- Other extensions you installed (and if the issue persists after disabling them):
- A clear and concise description of what the bug is.

**To Reproduce**
1. Go to register level (not bit field level).
2. Move the mouse over the register to show the tooltip.

The tooltip seems to be limited to a fixed range.
Longer text items like the 32-bit binary value, but also the `Decimal` heading see line breaks.

**Expected behavior**

Before the change, the tooltip box was wide enough to accommodate the above without line breaks. Width presumably dynamically calculated?

**Code sample and logs**
N/A, use any example

**Screenshots**

Before PrimeReact introduction
![image](https://github.com/user-attachments/assets/fe96611e-f36a-4d2e-bafe-b2b6f4759020)

After PrimeReact introduction
![image](https://github.com/user-attachments/assets/a40f25bd-42aa-4811-bb09-d39e545a3b26)

**Additional context**

**Comment by martin-fleck-at** [comment]
I believe that has much more to do with the switch from the Tree View API in VS Code to the Web View API. A web view is rendered in an iframe, making the tooltip adhere to the bounds of the iframe. We could make the tooltip bigger but that would introduce scrollbars in our tree and not actually overlap with the other views in VS Code. Unfortunately, I see no way to access the parent window from inside an iframe since VS Code very deliberately [removes access to that when loading webviews](https://github.com/microsoft/vscode/blob/1d4130b8b39bf3bcbc933eaae2f3fb366530ecc2/src/vs/workbench/contrib/webview/browser/pre/index.html#L223) so attaching the tooltip to the parent window is also not an option. Given that this is a choice made by VS Code, maybe restyling the tooltip a bit to reduce the number of breaks would be a better option, what do you think?

**Comment by thegecko** [comment]
Yeah, you are restricted to the view iframe when using webviews :(
